Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Iterative_authalic_parameterizer_3.h

```diff
@@ -632,41 +631,33 @@ class Iterative_authalic_parameterizer_3
   /// \param mesh a triangulated surface.
   /// \param main_vertex_v_i the vertex of `mesh` with index `i`
   /// \param neighbor_vertex_v_j the vertex of `mesh` with index `j`
-  NT compute_w_ij(const Triangle_mesh& tmesh,
+  NT compute_w_ij(const Triangle_mesh& mesh,
```

Comment:
The t of tmesh stands for triangle, so not really a need to make this change.

Reply:
I changed it to be consistent with the naming used inside this function for other parameterizers. Even, in the doxygen above, you can see that they use `mesh` instead of `tmesh`. But I reverted it back to `tmesh` since it is probably more correct in this context. You are right.